### DISCUSSION TITLE
[Feature Request] Drop file area

### DISCUSSION THREAD

**Discussion by Sinan-Karakaya** [heroui-inc/heroui#1242] [Feature requests]
### Is your feature request related to a problem? Please describe.

Almost no CSS for a file type Input component

### Describe the solution you'd like

A DropFileArea Component, with possible preview of file dropped, file type filter...

### Describe alternatives you've considered

Better default CSS for the Input file type component

**Comment by babblebey** [comment]
I think this will need some design to get started with!

**Comment by wingkwong** [comment]
let's track in https://github.com/heroui-inc/heroui/discussions/1814
